HLTrigger/Configuration/python/customizeHLTforCMSSW.py

```diff
@@ -404,6 +404,16 @@ def customizeHLTfor44778(process):
         if hasattr(process, modLabel):
             mod = getattr(process, modLabel)
             mod.l1EGCand = cms.InputTag('hltEgammaCandidatesPPOnAA')
+
+def customizeHLTFor45478(process):
+
+    ca_builders = ["CAHitNtupletAlpakaPhase1@alpaka", "alpaka_serial_sync::CAHitNtupletAlpakaPhase1"]
+
+    for ca_builder in ca_builders:
+        for producer_name in producers_by_type(process, ca_builder):
+            producer = getattr(process,producer_name.label_())
+            producer.phiCuts = cms.vint32( 522, 730, 730, 522, 626, 626, 522, 522, 626, 626, 626, 522, 522, 522, 522, 522, 522, 522, 522 , 522, 522)
+
```

Comment:
Can you please remove this change of HLTrigger/Configuration/python/customizeHLTforCMSSW.py and instead file a CMSHLT JIRA ticket for this HLT config chnage? Assuming this can be separated?

Reply:
For the ticket I'd leave @lguzzi and @elusian. For the customizer, the problem of removing this is that it would break any step2 running 2024 HLT (and also 2023 with CUDA modules), no?

Reply:
Hmm, that is a pity: typically it is better to have the PR adding functionality (but leaving as 'default' the behaviour as before), and then in a second step making the config change to actually use the new functionality.

Reply:
You could use `fillDescriptions` to set the default values and therefore can have the old behavior.

Reply:
Not in this case as this is not a newly-added parameter, but one already set and now changed.